### DISCUSSION TITLE
Any chance to get titled activities

### DISCUSSION THREAD

**Discussion by Ahrron** [mytourbook/mytourbook#1387] [General]
I have my activities named in the format
2021-10-15_145223_NW_Jammertal_5_activity_7661248301
YYYY-MM-DD_HHMMSS_Activity-Name_activity_ActivityID.
After import the title of the imported activity is always empty.
If I name the activity in myTourbook and export it to tcx, the title is included in the tcx file in the tag [**name**]. 
But if I include [**Name**]Activity-Name[/**Name**] in the tcx files, it is ignored and the title of the activity remains empty.

How to get activity name/file name into imported activities?

[ = {

**Comment by FJBDev** [comment]
@Ahrron 
Can you provide a file example to reproduce what you describe ?

**Comment by Ahrron** [comment]
FJBDev ***@***.***> schrieb am Mi., 7. Aug. 2024, 02:02:
